lib/schema.js

```diff
@@ -400,21 +399,17 @@ Schema.prototype.defaultOptions = function(options) {
   const baseOptions = get(this, 'base.options', {});
   options = utils.options({
     strict: 'strict' in baseOptions ? baseOptions.strict : true,
-    strictQuery: 'strictQuery' in baseOptions ? baseOptions.strictQuery : false,
     bufferCommands: true,
     capped: false, // { size, max, autoIndexId }
     versionKey: '__v',
     optimisticConcurrency: false,
-    discriminatorKey: '__t',
```

Comment:
This change also seems unintentional. Deleting the `discriminatorKey` option is most definitely a non-goal